lib/monit_interface.py

```diff
@@ -127,3 +134,46 @@ def restart(watch):
 
   logging.info("Restarting watch {0}".format(watch))
   return run_with_retry([MONIT, 'restart', '-g', watch])
+
+def create_monit_config_file(watch, start_cmd, stop_cmd, ports, match_cmd,
```

Comment:
Okay, I would recommend trying to adjust the behavior of the function with parameters if possible. If not, I would rename this function so that it's not used as a general way to create a monit config file.